### PR TITLE
Add readable secondaries and sync secondary to commit to SQL MIAA create

### PR DESCRIPTION
**Pull request by candiceye**
Changes:
- Added arguments 'readable secondaries' and 'sync secondary to commit' to SQL MIAA create in notebook 'az sql mi-arc create' command
- Added dynamically changing radio fields for SQL MIAA create for 'readable secondaries' and 'sync secondary to commit'
- Added dynamically updating field for readable secondaries to the SQL MIAA cost summary section
- Added an editable field for "synchronized secondaries" to the SQL MIAA dashboard edit page
image
image

In the future:
- Will change those fields from radio to dropdown
- Will change the dynamic values for the replicas to be generated by a valueprovider function rather than hard-coded

### CODE REVIEW

File: extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts [hunk 1]
```diff
@@ -51,7 +52,7 @@ export class RadioGroupLoadingComponentBuilder implements azdata.ComponentBuilde
 					label: option.displayName,
 					value: option.name,
 					checked: option.displayName === defaultValue,
-					enabled: instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? false : this._fieldInfo.enabled // Dynamic enablement is initially set to false
+					enabled: instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? this._enabled : this._fieldInfo.enabled // Dynamic enablement is initially set to false
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts:55]: Please do the fixes here as a separate PR

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts (hunk 1 above)
Comment (candiceye) [reply extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts]: Done

**Review by Charles-Gagnon — APPROVED** [review]
